cumulusci/salesforce_api/tests/test_metadata.py

```diff
@@ -224,7 +224,7 @@ def test_call_faultcode(self):
         response = '<?xml version="1.0" encoding="UTF-8"?><faultcode>foo</faultcode>'
         self._mock_call_mdapi(api, response)
         with self.assertRaises(MetadataApiError):
-            resp = api()
+            resp = api()  # noqa: F841
```

Comment:
I'd say just don't assign the variable if we're not using it (here and below).

```suggestion
            api()
```